### PR TITLE
release: Tag and fork documentation repo as part of release

### LINKED ISSUE #1007: Tag and fork documentation repo as part of release
See : https://github.com/kata-containers/documentation/issues/246

### PR DESCRIPTION
**Pull request by amshinde**
We should start maintaining stable branches for the documentation
repo similar to other repos.

Fixes #1007

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

**Comment by amshinde** [comment]
@jcvenegas Please take a look.

File: release/tag_repos.sh [hunk 1]
```diff
@@ -76,6 +76,7 @@ repos=(
 	"runtime"
 	"shim"
 	"tests"
+	"documentation"
```
Comment (jodh-intel) [inline release/tag_repos.sh:79]: Oh - extra points for adding this in its alphabetical location ;)

**Comment by chavafg** [comment]
Seems like we need to tag documentation before merging this: 
```
17:39:35 Check tag_repos.sh status
17:39:36 ERROR: documentation is not in version 1.11.0-alpha1
```

File: release/tag_repos.sh (hunk 1 above)
Comment (amshinde) [reply release/tag_repos.sh]: @jodh-intel Fixed.

**Comment by amshinde** [comment]
> Seems like we need to tag documentation before merging this:

Ok. Should we tag a commit around the time we made 1.11.0-alpha1 release.

**Comment by jodh-intel** [comment]
Restarted snap CI which had timed out.

**Pull request merged**